test/src/main/java/io/strimzi/test/k8s/KubeClient.java

```diff
@@ -91,6 +91,10 @@ public void createNamespace(String name) {
         client.namespaces().resource(ns).createOrReplace();
     }
 
+    public void createNamespace(final Namespace namespace) {
```

Comment:
Should it be called `createOrReplaceNamespace` then?

Reply:
> Should it be called `createOrReplaceNamespace` then?

Yes, will do.